geoapps/io/InducedPolarization/constants.py

```diff
@@ -218,6 +220,8 @@
             if default_ui_json[k]["isValue"] == False:
                 key = "property"
         default_ui_json[k][key] = v
+        if "enabled" in default_ui_json[k].keys() and v is not None:
```

Comment:
I don't think this will work with geopy-318.  There will be times where we want the value to be not None with enabled false.  See the none_map feature in 318.

Reply:
Done. Just need to wait for the tests to pass